src/tools/PICS-generator/PICSGenerator.py

```diff
@@ -53,6 +53,9 @@ def GenerateDevicePicsXmlFiles(clusterName, clusterPicsCode, featurePicsList, at
     elif "Group Key Management" == clusterName:
         clusterName = "Group Communication"
 
```

Comment:
```suggestion
```

If you want we can just remove this one then. The names changes from time to time.

Reply:
This was tested, so I am leaving it :)

Reply:
No problem, we can always clean up later.